LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/email-set-value-expected.txt

```diff
@@ -1,4 +1,4 @@
 
 
-PASS setValue(sanitizedValue) is reflected in visible text field content
+FAIL setValue(sanitizedValue) is reflected in visible text field content assert_equals: visible value is unsanitized expected " foo@bar   " but got ""
```

Comment:
Should we be changing the test expectation here? You say the new behavior matches Firefox. Why leave WPT expecting the wrong thing? Should the test be more lenient? Or should it be fixed?

Reply:
It seems like we can fix the test to not depend on this old WebKit quirk without waiting for engines to change. I don’t think the test is intended to test what `getSelection().toString()` does. I suppose it’s OK to have a test for that, but this test is supposed to check how the sanitizing of that element works, which we should find another way to test, right?

Reply:
The test is checking what the visible value of the text is (it's not sanitized). I'm not aware of any other way to do that so "fixing" the test would basically mean getting rid of these checks.

Reply:
Maybe copy and paste sees this “visible” value?

Reply:
I guess one way to test it is turn it into a ref test.

Reply:
Alright, posted a WPT PR here to convert it to a ref test: https://github.com/web-platform-tests/wpt/pull/37773